app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,15 @@
+class MechanicsController < ApplicationController
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def update
+    @mechanic = Mechanic.find(params[:id])
+    # :add_ride grabs what's entered into text have_field
+    # params brings it along
+    ride = Ride.find(params[:add_ride])
+    RideMechanic.create!(ride: ride, mechanic: @mechanic)
```

Comment:
This works, or because you're inthe Mechanics controller Update action, you could do `@mechanic.rides << ride` and that would essentially do the same thing.